db/migrations/0009_add_wallet_details_table.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS wallet_details (
+  address TEXT PRIMARY KEY,
+  is_sanctioned BOOLEAN
+);
+
+-- Backfill existing wallet addresses
```

Comment:
This shouldn't be necessary, as also need to insert rows when we first check their sanctions status. But, it also doesn't hurt

Reply:
I should include that into a separate PR.